### DISCUSSION TITLE
Display all results to filter, empty search

### DISCUSSION THREAD

**Discussion by arturamiryan** [Pagefind/pagefind#483] [Q&A] [answered]
Hello, is there any possibility for displaying all results alongside with filters with empty search? I know that there is exist empty search API for js, I can just push null. But I really don't want to rewrite all the ui by myself :)

**Comment by bglw** [comment, marked as answer]
Hello! 👋 

Not with the default UI at the moment, though the [modular UI](https://www.npmjs.com/package/@pagefind/modular-ui) would be closer to providing what you need. It's somewhere between the all-in-one UI package and writing your own, as it helps with a lot of the work connecting the components together and managing search state.
